specs/DotNetProjectFile.Analyzers.Specs/Rules/MS_Build/Correct_spelling_of_nodes.cs

```diff
@@ -0,0 +1,47 @@
+using DotNetProjectFile.MsBuild;
+using System.IO;
+
+namespace Rules.MS_Build.Correct_spelling_of_nodes;
+
+public class Reports
+{
+    [Test]
+    public void on_spelling_errors() => new CorrectSpellingOfNodes()
+        .ForProject("Misspelled.cs")
+        .HasIssues(
+            new Issue("Proj0031", "The node name EnableNetAnalyzers is misspelled, EnableNETAnalyzers might be intended."),
```

Comment:
I feel like this message might be a bit too aggressive, I would maybe go for something along the lines of how Microsoft Office Word phrases it